typings/index.d.ts

```diff
@@ -675,7 +675,7 @@ declare module 'klasa' {
 		private _prompted: number;
 		private _currentUsage: Tag;
 
-		public run<T = any[]>(prompt: string): Promise<T>;
+		public run<T = any[]>(prompt: string | MessageEmbed): Promise<T>;
```

Comment:
Might as well need to import `MessageAdditions` and `APIMessage`, as well as updating the [docs](https://github.com/dirigeants/klasa/blob/0e00e4c36729e701f437660e15e91e65a57d9219/src/lib/usage/TextPrompt.js#L165):

```suggestion
		public run<T = any[]>(prompt: StringResolvable | MessageOptions | MessageAdditions | APIMessage): Promise<T>;
```